types/allure-js-commons/allure-js-commons-tests.ts

```diff
@@ -0,0 +1,24 @@
+import * as Allure from "allure-js-commons";
```

Comment:
Since this is a constructable entity, it isn't legal in ES2015 to import this with a namespace import like `import * as foo from 'foo'`, so could you please convert this to either

```ts
import foo = require('foo')
```

or the following with `esModuleInterop` turned on

```ts
import foo from 'foo';
```

Reply:
done